webapp/templates/public/homepage.html.twig

```diff
@@ -0,0 +1,60 @@
+{% extends "public/base.html.twig" %}
+
+{% block title %}Korespondenční seminář{% endblock %}
+{% block content %}
```

Comment:
zkus tomu dat alespon basic styly, dat to do containeru nebo vyuzit nejake komponenty z bootstrapu, ktery DJ pouziva

Reply:
container-fluid tam jiz je z /templates/base.
zkusim si pohrat s marginem a treba pouzit cards.

Reply:
aha, myslel jsem, ze do containeru se dalsi container nedava, ale jo. :)
takze pridam.